app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -58,6 +59,15 @@ def dest_resource_pool
     cluster ? cluster.default_resource_pool : dest_host.default_resource_pool
   end
 
+  def dest_storage_profile
+    # Storage Profiles were added in vSphere 5.5 so make sure we
+    # don't send newer api types to an older vCenter
+    return nil if source.ext_management_system.api_version < '5.5'
```

Comment:
Is this same api version check done when populating the list of available storage profiles? (I have no idea where that's actually done, or I'd check for myself :smirk_cat:)